packages/vb-decompiler-lite.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,9 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'VB Decompiler Lite'
+$category = 'Visual Basic'
+
+VM-Uninstall-With-Uninstaller $toolName $category "EXE" "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-"
+
+VM-Uninstall $toolName $category
```

Comment:
Why is this function call needed? Is there something to remove after executing `VM-Uninstall-With-Uninstaller`? I have noticed that some similar packages call both, but I suspect this is wrong.

Have you tested the uninstallation as well?

Reply:
I haven't tested that, I saw it in other packages

Reply:
You should also test the uninstallation locally (as we do not test it in the CI). After testing the installation, run `choco uninstall <package_name>` and ensure the uninstallation runs without error and that the tool and the shortcut are deleted. :wink:

Reply:
I have run the uninstallation locally:
![image](https://github.com/user-attachments/assets/22fe0100-db31-46b0-a081-a6f598808fd5)

I get a message box although we are specifying `/SUPPRESSMSGBOXES`. I don't understand why this happens. As uninstallation is best-effort, I do not want to block this PR because of that. @sara-rn could you open an issue to document this issue?